src/main/java/dan200/computercraft/ComputerCraft.java

```diff
@@ -277,6 +281,9 @@ public void preInit( FMLPreInitializationEvent event )
         Config.turtlesCanPush = Config.config.get( Configuration.CATEGORY_GENERAL, "turtlesCanPush", turtlesCanPush );
         Config.turtlesCanPush.setComment( "If set to true, Turtles will push entities out of the way instead of stopping if there is space to do so" );
 
+        Config.minTimeBetweenSounds = Config.config.get( Configuration.CATEGORY_GENERAL, "minTimeBetweenSounds", minTimeBetweenSounds);
+        Config.minTimeBetweenSounds.setComment("The minimum time in between calls to sound.play on one computer in milliseconds" );
```

Comment:
That will register the language key, but you haven't actually provided a translation - if you open the GUI, you'll just see `minTimeBetweenSounds`. If you add the appropriate string to `en_US.lang`, then it should be translated correctly.